Makefile

```diff
@@ -231,10 +231,11 @@ verify: .init .generate_files
 	@$(DOCKER_CMD) sh -c \
 	  'for i in $$(find $(TOP_SRC_DIRS) -name *.go \
 	    | grep -v generated \
+	    | grep -v internal_interfaces.go \
```

Comment:
rule?

everything in pkg/client is generated. There's no point in checking any of it.